macros/PGbasicmacros.pl

```diff
@@ -103,6 +105,8 @@ sub _PGbasicmacros_init {
 	# initialize the my variable version of $displayMode from the "runtime" version
 	# of main::displayMode
 
+	# dd "in _PGbasicmacros_init";
+
```

Comment:
This looks like a remnant of some local debugging.  This should be deleted.

Reply:
It looks like the only file that needs perltidying is t/build_PG_envir.pl.

Reply:
fix.